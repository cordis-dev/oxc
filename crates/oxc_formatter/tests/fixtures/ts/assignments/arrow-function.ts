{
  const onPanning: ComponenASDtProps<
    typeof TransformWrapper
  >["onPanning"] = () => {
  }
}
const onPanning: ComponenASDtProps<
  typeof TransformWrapper
>["onPanning"] = () => {
}
