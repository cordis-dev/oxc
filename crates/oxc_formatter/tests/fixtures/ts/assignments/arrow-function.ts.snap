---
source: crates/oxc_formatter/tests/fixtures/mod.rs
---
==================== Input ====================
{
  const onPanning: ComponenASDtProps<
    typeof TransformWrapper
  >["onPanning"] = () => {
  }
}
const onPanning: ComponenASDtProps<
  typeof TransformWrapper
>["onPanning"] = () => {
}

==================== Output ====================
{
  const onPanning: ComponenASDtProps<
    typeof TransformWrapper
  >["onPanning"] = () => {};
}
const onPanning: ComponenASDtProps<
  typeof TransformWrapper
>["onPanning"] = () => {};

===================== End =====================
